e2e/lookout_ingester_test/lookout_ingester_test.go

```diff
@@ -442,7 +442,10 @@ func createTestTriggers(db *sql.DB) {
 func dropTestTriggers(db *sql.DB) {
 	for _, trigger := range testTriggers {
 		// We don't care about the result. Just drop them.
-		db.Exec(fmt.Sprintf("DROP TRIGGER %s on %s", trigger.Name(), trigger.Table))
+		_, err := db.Exec(fmt.Sprintf("DROP TRIGGER %s on %s", trigger.Name(), trigger.Table))
```

Comment:
@ClifHouck I tried to panic as you do in these tests but it failed the test.  Is there anything bad happening that this shows?  Should we be getting errors during dropping triggers in the e2e tests?

Reply:
I realized the 'right' way to deal with this is to add a `IF EXISTS` clause to these statements. I can fix this in another PR or push a commit to your branch (although I don't know if I have permission to do this).

Reply:
Addressed as part of https://github.com/G-Research/armada/pull/1889